Revamp of the 'Team' page (now renamed 'Workgroups')

erciccione opened the pull request:
Closes #895 

This changes completely the structure of the "Team" page, now renamed "Workgroups".

Instead of single members we show a list of the main workgroups, a description and brief instructions to start to contributing.

~~This PR is a draft because needs some minor edits and to give the opportunity to community member to improve descriptions and "where to start" sections~~. Feel free to suggest better wording, different instructions or new/different workgroups.


Preview:
![Screenshot_2020-11-28  Workgroups ](https://user-images.githubusercontent.com/28106476/100517012-a2fc8d80-317f-11eb-9ea7-69a36d2c2a8c.png)

xmrdsc commented:
Perhaps instead of `<i>` for the IRC channel names `<code>` looks better;

![https://i.imgur.com/jaiEMdS.png](https://i.imgur.com/jaiEMdS.png)

SamsungGalaxyPlayer commented:
Not to get too meta here, but what will the Hangouts page turn into? What's the difference between that page and this one? If a workgroup makes a "Hangout" like a forum or an IRC channel, how will that be handled?

---
Monero Space

Monero Space is an active, project-focused Monero workgroup that provides services to the Monero community. It hosts popular resources and organizes events such as Monero Meet.

---

Malware Response Workgroup

The Monero Malware Response Workgroup provides resources and live support for multiple types of Monero-related malware.

#monero-mrw 

https://mrw.getmonero.org

ThomasBucaioni commented:
> Perhaps instead of `<i>` for the IRC channel names `<code>` looks better;
Maybe just a Courier typeface?

erciccione commented:
@SamsungGalaxyPlayer 

> but what will the Hangouts page turn into?

I consider the hangouts page as a simple list of chatrooms/places where community members can be found to discuss different topics. I see how the two can be seen as overlapping, but i don't think that's the case. For example `#monero-offtopic` makes sense in hangouts, but not in the workgroups page.

> workgroup makes a "Hangout" like a forum or an IRC channel, how will that be handled?

if it's a resource of the workgroup should be added in the entry of the workgroup (for eample `forum: link`), but could also be added to the hangouts page.

> Malware Response Workgroup

do you have a suggestion for the "Where to start" field?

erciccione commented:
> Perhaps instead of <i> for the IRC channel names <code> looks better;

I considered it initially, but then avoided it because the highlighted text is slightly bigger on firefox:

![Screenshot from 2020-10-20 15-25-07](https://user-images.githubusercontent.com/28106476/96600011-66b85080-12e8-11eb-9b58-a79816a692b1.png)

but if people prefer it i'll just make the change.

SamsungGalaxyPlayer commented:
@erciccione Where to start: Join the channel and ask questions or provide support.

erciccione commented:
- Replaced `<i>` with `<code>`
- Edited description of monero space
- Added entry for "Malware Response Workgroup"
- Updated screenshot
- Marked 'Ready for review'

SamsungGalaxyPlayer commented on _i18n/en.yml:
```diff
@@ -124,14 +124,42 @@ sponsorships:
+  community_descr: Workgroup used by the community to organize and coordinate. This is the place where every non-coder who wants to contribute to Monero should start.
+  community_start: The best way to start contributing to this workgroup is to join the chatrooms and introduce yourself, stating your interests and the skills you would like to offer to the community.
+  devworkgroup: Development Workgroup
+  devworkgroup_descr: This workgroup reunites Monero developers working on the wallets and daemon maintained by the Core Team. They are mostly divided in GUI and CLI devs and they coordinate using two different chatrooms.
```
"in GUI and CLI devs" is awkward wording. Maybe change the last sentence to "This workgorup consists of two main parts: the CLI and the GUI teams. These teams coordinate using two different chatrooms."

SamsungGalaxyPlayer commented on _i18n/en.yml:
```diff
@@ -124,14 +124,42 @@ sponsorships:
+  web_descr: Another dev-focused workgroup where to coordinate the development of the website. Most devs in there are part of the development workgroup. The website workgroup was created mostly to have discussions related to the website all in one place.
+  web_start: It's always a good idea to join the chatroom and talk directly with the members of the workgroup, but if you prefer to get started by yourself, take a look at the readme in the GitHub repository. It contains step by step instructions to edit every single part of this website.
+  localization: Localization Workgroup
+  localization_descr: One of the oldest Monero Workgroups. It's formed by contributors working on translating Monero-related software and documentation. It's strongly suggested to contact the workgroup before working on translations.
```
Recommend removing first sentence, then changing the beginning of the second sentence to "This workgroup consists of contributors who translate..."

erciccione replied:
Why removing the first sentence? it's an anecdotal reference, but at least makes the "description" field less monotone

SamsungGalaxyPlayer replied:
Just because it doesn't describe anything relevant/actionable to the audience imo.

erciccione replied:
I disagree. The localization workgroup is the oldest organized workgroup of the community. IMO it's relevant to mention it in a page dedicated to Monero workgroups and it's also something that makes me proud. It's a description, doesn't need to be actionable.

erciccione commented:
Rebased and added suggestions.

binaryFate commented on _i18n/en.yml:
```diff
@@ -141,14 +140,42 @@ sponsorships:
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
```
core_descr: Stewards of the community, the Core Team manages and maintains infrastructure where centralization cannot be avoided (domains, website, github repositories...). Its members act as mediators as they see fit and aim to weigh consensus in the community, perpetuating the quality interactions that made Monero the successful project it is today. The Core Team does not and has never received any financial compensation.

binaryFate replied:
Note this text was agreed with the others

erciccione replied:
Updated

binaryFate commented:
Suggested new text for core team. (Agreed with the others).

binaryFate commented:
@erciccione shall we proceed or are you waiting for some more feedback? I think it's a cool PR I want to see it out :)

erciccione commented:
@binaryFate i'm waiting for reviews/approvals :)

SamsungGalaxyPlayer commented:
Would it be possible to add the following for Monero Space: 

Matrix: https://matrix.to/#/#monero-space:matrix.org

Flarum: https://forum.monero.space (with the [Flarum logo?](https://opencollective-production.s3.us-west-1.amazonaws.com/0959f4e0-0559-11ea-97cf-5fc2f9568174.png))

Website: https://monero.space

erciccione commented:
> Would it be possible to add the following for Monero Space:

Added (except the Flarum logo) and updated the preview

hyc commented on _i18n/en.yml:
```diff
@@ -141,14 +140,42 @@ sponsorships:
+  outreach_descr: This workgroup is focused on getting Monero out there, gathering artists, writers and anyone who wish to make Monero known to the World.
+  outreach_start: The members of the Outreach workgroup heavily use Telegram for coordination and matrix/freenode in minor part. If you want to help, you have to contact the workgroup on one of this channels. They are quite structured, so you'll probably be directed to a coordinator.
+  telegram: "Telegram chat:"
+  mrl_descr: As the name suggests, the Monero Research Lab is the workgroup focused on Monero related research. It's formed by scientists trying to find a way to improve Monero's protocol and features. Most of the protocol improvements who make Monero amazing, like CLSAG and @Bulletproofs, are the result of the work of this workgroup.
```
I know this PR has already been merged, but this should have been:
s/find a way/find ways/   <-- are we really only looking for one way to improve? nope.
s/improvements who make/improvements that make/   <-- the improvements are things, not people.

erciccione replied:
#1378
